tfe/resource_tfe_organization_membership.go

```diff
@@ -108,3 +110,25 @@ func resourceTFEOrganizationMembershipDelete(d *schema.ResourceData, meta interf
 
 	return nil
 }
+
+func resourceTFEOrganizationMembershipImporter(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	tfeClient := meta.(*tfe.Client)
```

Comment:
Panic :exclamation: 

```suggestion
	config := meta.(ConfiguredClient)
	// you can use the tfe client: config.Client.Workspaces.Blah()
```
This change was introduced in #762

Reply:
Fixed! Thanks for catching that